### ISSUE TITLE
Multiple photos not working

### ISSUE THREAD

**Issue by rameshbansal** [plangrid/react-file-viewer#87]

**Comment by TheThirdDave** [comment]
I ran into this issue as well and think I may have a fix for it.

Was looking into the code and noticed the driver uses `document.getElementById` to append the images to a `<div>` with the id `pg-photo-container`. Anyone else have any thoughts?

**Comment by TheThirdDave** [comment]
Yes! Sorry haven't had a chance to get back to this but I did get the fix working for this.

Haven't made a pull request yet for it though. The fix though is replacing uses of `document.getElementById` with React References instead.

**Comment by macrozone** [comment]
yeah, using ids is a big no-go in javascript and in particular in react. Has anyone forked and fixed that? should be really easy though

**Comment by thefedoration** [comment]
This PR addresses this issue: https://github.com/plangrid/react-file-viewer/pull/119
Can we review it please?

**Comment by macrozone** [comment]
@thefedoration unfortunalty plangrid abondoned this package

**Comment by kgdiem** [comment]
@michaelwolo  Are you open to new maintainers?
